playground/frontend/playground_components/lib/widgets/split_view.dart

```diff
@@ -1,261 +0,0 @@
-/*
```

Comment:
You can use `git mv source_file_name destination_file_name` to move files so that `git` tracks it. Otherwise it does its best to guess if a file was deleted or moved, and often fails like in this case.